pkg/logs/config/endpoints.go

```diff
@@ -93,6 +93,15 @@ func (e *Endpoint) GetStatus(prefix string, useHTTP bool) string {
 	return fmt.Sprintf("%sSending %s logs in %s to %s on port %d", prefix, compression, protocol, host, port)
 }
 
+func (e *Endpoint) GetIsReliable() bool {
+	return e.IsReliable == nil || *e.IsReliable
+}
+
+func (e *Endpoint) SetIsReliable(reliable bool) {
+	val := reliable
+	e.IsReliable = &val
+}
+
```

Comment:
this appears unused?

Reply:
oops yep. leftover from a previous revision.